### PR TITLE
style(apps): update the style of application list

### PR DESCRIPTION
**Pull request by thomas-mauran**
### Preview
image

### Description

Linked to issue #179 

* Adding categories
* New applications from the original tui-rs repo

### CODE REVIEW

File: APPS.md [hunk 1]
```diff
@@ -1,48 +1,86 @@
...
-| [vector](https://vector.dev)                                         | A lightweight, ultra-fast tool for building observability pipelines                                       | [vectordotdev](https://github.com/vectordotdev)          |
-| [xplr](https://github.com/sayanarijit/xplr)                          | A hackable, minimal, fast TUI file explorer                                                               | [Arijit Basu](https://github.com/sayanarijit/xplr)       |
-| [ytop](https://github.com/cjbassi/ytop)                              | A TUI system monitor written in Rust (no longer maintained)                                               | [Caleb Bassi](https://github.com/cjbassi)                |
-| [zenith](https://github.com/bvaisvil/zenith)                         | Sort of like top or htop but with zoom-able charts, CPU, GPU, network, and disk usage                     | [Benjamin Vaisvil](https://github.com/bvaisvil)          |
+#### 💻 Development Tools:
```
Comment (orhun) [inline APPS.md:3]: I suggest we drop the ":' at the end of the titles.

File: APPS.md [hunk 2]
```diff
@@ -1,48 +1,86 @@
 # Apps using `ratatui`
```
Comment (orhun) [inline APPS.md:1]: How about also adding a short description and table of contents?

File: APPS.md [hunk 3]
```diff
@@ -1,48 +1,86 @@
...
+| [oxker](https://github.com/mrjackwills/oxker)            | Simple TUI to view & control docker containers.                                              | [mrjackwills](https://github.com/mrjackwills)   |
+| [systeroid](https://github.com/orhun/systeroid)          | A more powerful alternative to sysctl(8) with a terminal user interface.                     | [orhun](https://github.com/orhun)               |
+| [xplr](https://github.com/sayanarijit/xplr)              | Hackable, minimal, and fast TUI file explorer.                                               | [sayanarijit](https://github.com/sayanarijit)   |
+| [ytop](https://github.com/cjbassi/ytop)                  | TUI system monitor for Linux.                                                                | [cjbassi](https://github.com/cjbassi)           |
+| [zenith](https://github.com/bvaisvil/zenith)             | Cross-platform monitoring tool for system stats.                                             | [bvaisvil](https://github.com/bvaisvil)         |
+
+#### 🌌 Other Categories:
```
Comment (orhun) [inline APPS.md:76]: Let's rename this to "Other"

File: APPS.md (hunk 1 above)
Comment (thomas-mauran) [reply APPS.md:3]: done :+1:

File: APPS.md (hunk 3 above)
Comment (thomas-mauran) [reply APPS.md]: done :+1:

**Comment by joshka** [comment]
I like what you're doing here. Happy to approve it, but  let's drop the authors and move to this just being markdown lists instead of tables.

**Review by joshka — APPROVED** [review]

**Comment by thomas-mauran** [comment]
> I like what you're doing here. Happy to approve it, but let's drop the authors and move to this just being markdown lists instead of tables.

done :+1:

**Comment by orhun** [comment]
Did some tweaks about formatting in [baae4b8](https://github.com/tui-rs-revival/ratatui/pull/184/commits/baae4b8d4d04cf7eefbcd9efca585594e6ad3792) and added the missing description in [3e4d1c8](https://github.com/tui-rs-revival/ratatui/pull/184/commits/3e4d1c89e81dbf0cfead8c38497833c15ed988f0)

Thank you for your contribution!

**Review by orhun — APPROVED** [review]

**Review by sayanarijit — APPROVED** [review]

**Pull request merged**